PWGHF/HFC/DataModel/CorrelationTables.h

```diff
@@ -200,6 +205,64 @@ DECLARE_SOA_TABLE(DplusHadronRecoInfo, "AOD", "DPLUSHRECOINFO", //! D+-Hadrons p
                   aod::hf_correlation_dplus_hadron::MD,
                   aod::hf_correlation_dplus_hadron::SignalStatus);
 
+// definition of columns and tables for Dstar-Hadron correlation pair
+using Dstars = HfCandDstar;
+using Dstar = Dstars::iterator;
+namespace hf_correlation_dstar_hadron
+{
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);
+// Dstar candidate properties
+DECLARE_SOA_INDEX_COLUMN(Dstar, dstar);
+DECLARE_SOA_COLUMN(PhiDstar, phiDstar, float);
+DECLARE_SOA_COLUMN(EtaDstar, etaDstar, float);
+DECLARE_SOA_COLUMN(PtDstar, ptDstar, float);
+DECLARE_SOA_COLUMN(MDstar, mDstar, float);
+// DECLARE_SOA_COLUMN(IsPrompt,isPrompt,bool); // although this also defined in (HfCandDstarMcRec HfCandDstarMcRec) tables
+// DECLARE_SOA_COLUMN(MatchingStatus, matchingStatus, bool); // although this also defined in (HfCandDstarMcRec HfCandDstarMcRec) tables
+// Track properties
+DECLARE_SOA_INDEX_COLUMN(Track, track);
+DECLARE_SOA_COLUMN(PhiTrack, phiTrack, float);
+DECLARE_SOA_COLUMN(EtaTrack, etaTrack, float);
+DECLARE_SOA_COLUMN(PtTrack, ptTrack, float);
+// common
+DECLARE_SOA_COLUMN(TimeStamp, timeStamp, int64_t);
+DECLARE_SOA_COLUMN(PoolBin, poolBin, int);
+// Dynamic columns
+DECLARE_SOA_DYNAMIC_COLUMN(DeltaEta, deltaEta, [](float etaTrack, float etaCandidate) -> float { return (etaTrack - etaCandidate); });
+DECLARE_SOA_DYNAMIC_COLUMN(DeltaPhi, deltaPhi, [](float phiCandidate, float phiTrack) -> float { return RecoDecay::constrainAngle(phiTrack, phiCandidate); });
```

Comment:
```suggestion
DECLARE_SOA_DYNAMIC_COLUMN(DeltaPhi, deltaPhi, [](float phiCandidate, float phiTrack) -> float { return RecoDecay::constrainAngle(phiTrack - phiCandidate); });
```

Reply:
/// Constrains angle to be within a range.
  /// \note Inspired by TVector2::Phi_0_2pi in ROOT.
  /// \param angle  angle
  /// \param min  minimum of the range
  /// \return value within [min, min + 2π).
  template <typename T, typename U = float>
  static T constrainAngle(T angle, U min = 0.)
  {
    while (angle < min) {
      angle += o2::constants::math::TwoPI;
    }
    while (angle >= min + o2::constants::math::TwoPI) {
      angle -= o2::constants::math::TwoPI;
    }
    return (T)angle;
  }
  

As per this definition, It accepts two argumnets.

Reply:
/// Constrains angle to be within a range.
  /// \note Inspired by TVector2::Phi_0_2pi in ROOT.
  /// \param angle  angle
  /// \param min  minimum of the range
  /// \return value within [min, min + 2π).
  template <typename T, typename U = float>
  static T constrainAngle(T angle, U min = 0.)
  {
    while (angle < min) {
      angle += o2::constants::math::TwoPI;
    }
    while (angle >= min + o2::constants::math::TwoPI) {
      angle -= o2::constants::math::TwoPI;
    }
    return (T)angle;
  }

Can we discuss on the definition of function ```constrainAngle(...)```? It accept two arguments, but I doubt if it really gives relative angle wrt candidate' s angle phi!

Reply:
"relative angle wrt candidate' s angle phi" = `phiTrack - phiCandidate`
The job of `constrainAngle` is to "Constrain the angle to be within a range".
What is your doubt?